### ISSUE TITLE
GE: deployed minio 9605 but not reachable

### ISSUE THREAD

**Issue by hteibler** [threefoldtech/home#743]
wg-config:
```
[Interface]
Address = 100.64.21.254/32
PrivateKey = Y+Vp2orAwU9G6wlZpy3epwGGQbd1jabJQG4Dg9nHPAw=
[Peer]
PublicKey = ZhjHExJgExCb1PgpD1mD9kMG7Quh3T8oEpJAxaclpT0=
AllowedIPs = 172.21.0.0/16, 100.64.21.0/32
PersistentKeepalive = 25
Endpoint = 185.69.166.242:8637
```

**Comment by zaibon** [comment]
https://explorer.grid.tf/explorer/reservations/9605
Looking at the reservation detail, it never deployed properly. There are problem in the reservation format and the result shows an error.

```json
"volume_id": "## tfgrid.workloads.reservation.create.1\n - reservation_id      : 9,603\n\n    ## tfgrid.workloads.reservation.escrow.1\n     - address             : GABSJJG73UP2BDLIJCN6QK6ZV7PLTGKTQ4K3Y2TBY6QRWREY772WWNNY\n     - asset               : FreeTFT:GCBGS5TFE2BPPUVY55ZPEMWWGR6CLQ7T6P46SOFGHXEBJ34MSP6HVEUT\n     - details             :\n        - farmer_id: 82,872    - total_amount: 5,300,696,800\n\n-1",
```
This is wrong. volume ID should be something like "9063-1"

If you generated this reservation yourself, you most probably have a bug in your script. Feel free to share it here so we can have a look.

**Comment by hteibler** [comment]
please have a look

[minio_v1_py.txt](https://github.com/threefoldtech/home/files/4676285/minio_v1_py.txt)

**Comment by zaibon** [comment]
```python
volume_id=f'{registered_reservation_volume}-{volume.workload_id}',
```

Should be

```python
volume_id=f'{registered_reservation_volume.reservation_id}-{volume.workload_id}',
```

**Comment by hteibler** [comment]
deployed now: 9612 and 9613
can not reach http://172.21.31.13:9000
can reach http://172.21.31.13:7681/

```
[-] execvp: No such file or directory
 [with 0 args]
[+] subprocess: started process, pid: 8, pty: 8
[+] http: /attach/32059840 - 100.64.21.254 (::ffff:100.64.21.254)
[+] routing_get_id: requesting to attach id <32059840>
[+] callback: tty: request id: 32059840
[+] callback: tty: established: /ws/32059840 - 100.64.21.254 (::ffff:100.64.21.254), clients: 1
[+] callback: client not initialized
[+] callback: client not initialized
[+] callback: client not initialized
[+] callback: client not initialized
[+] callback: tty: ws closed from 100.64.21.254 (::ffff:100.64.21.254), clients: 0
[+] http: /api/process/start - 100.64.21.254 (::ffff:100.64.21.254)
[+] api: starting process: /bin/bash [with 0 args]
[+] corex: ========================================
[+] corex: initializing subprocess
[+] corex: starting: /bin/bash
[+] corex: ========================================
[+] http: /api/processes - 100.64.21.254 (::ffff:100.64.21.254)
[+] http: /attach/32059488 - 100.64.21.254 (::ffff:100.64.21.254)
[+] routing_get_id: requesting to attach id <32059488>
```
but I see only 4 results in 9612,
